apps/framework-cli/src/cli/local_webserver.rs

```diff
@@ -533,20 +537,29 @@ lazy_static! {
     static ref MOOSE_INGEST_API_KEY: Option<String> = get_env_var("MOOSE_INGEST_API_KEY");
 }
 
-async fn check_authorization(auth_header: Option<&HeaderValue>, env_var: &Option<String>) -> bool {
-    match env_var {
-        None => true,
-        Some(key) => {
-            let bearer_token = auth_header.and_then(|header_value| {
-                header_value
-                    .to_str()
-                    .ok()
-                    .and_then(|header_str| header_str.strip_prefix("Bearer "))
-            });
-
-            validate_token(bearer_token, key).await
-        }
+async fn check_authorization(
+    auth_header: Option<&HeaderValue>,
+    api_key: &Option<String>,
+    jwt_config: &Option<JwtConfig>,
+) -> bool {
+    let bearer_token = auth_header
```

Comment:
This currently prefers api key. 
- Should it be reversed? 
- Do we want the second option to have a way to override? 
- Should we support both? E.g. `X-Moose-Api-Key: key` and `Authorization: Bearer jwt-token`

Reply:
I am guessing it should be reversed.

For 2 reasons:
1. When you add the `JWT` property type, it will add optionality/endpoint, and we can add the `enforce_on_all_ingest_apis` config parameter. So, token auth could be a fallback for an API.
2.  JWT is more secure than token auth, as such, it should have a preference